arena.php

```diff
@@ -216,7 +216,12 @@ function arenaFight(): bool {
             echo "<table class='table'><tr><th>Battle Results</th></tr>
 			<tr><td>You have been defeated.
 			</td></tr></table>";
+			$player->health = 5;
             $player->ai_losses++;
+			$player->location = $player->village_location;
+			$location = explode('.', $player->location);
+			$player->x = $location[0];
+			$player->y = $location[1];

```

Comment:
Let's make this an instance method on the User class like `$player->moveToVillage()`, since we commonly have to do it

Reply:
done and updated it in arena/battles